### ISSUE TITLE
Once new synApps6.3 docker image is built...

### ISSUE THREAD

**Issue by prjemian** [prjemian/epics-docker#74]
Once #73 is merged into main and the new docker image is built, move content from `synApps6.3` and replace into parent folder.

_Originally posted by @prjemian in https://github.com/prjemian/epics-docker/issues/73#issuecomment-2047757459_

**Comment by prjemian** [comment]
not relevant now
